.gitattributes

```diff
@@ -8,3 +8,7 @@ packages/*/test/* linguist-vendored
 # Go build fails with Windows line endings.
 *.go text eol=lf
 go.mod text eol=lf
+
+# Mark certain files as "binary" -- not diffable
+**/test/fixtures/**/git/**/* binary
+**/test/fixtures/**/git/**/* linguist-generated
```

Comment:
I'm not sure if we want `linguist-generated` here. We use `linguist-vendored` above to disable stats but `linguist-generated` is going to hide diffs. I think we would want to see diffs when reviewing PRs.

Reply:
I added this because the git directory copies in the fixture added a lot of noise in the PR and it's less likely to get detailed review. I'd be fine removing these, though.

Reply:
Maybe I missed it. Where is the "git directory copies in the fixture" happening?

Reply:
I'm happy to remove this, but @EndangeredMassa and I added this on a call together, so it may be worth first discussing & getting on the same page about the best way to represent the `git` test files.

Reply:
Each test within the `createGitMeta()` tests starts by renaming `git` to `.git` and ends by renaming `.git` to `git`

Reply:
I see. Maybe that pattern should be added to `.gitignore` instead so the generated files are never committed?

Reply:
We _want_ the git files committed, since the tests operate on them. But Git doesn't like adding `.git` directories to the git repo (it tries to add them as git submodules instead), but in this case we want to commit those files, hence renaming them to something other than `.git`.

Reply:
Ok I understand now. This looks good since the diff is still there in the PR 

<img width="306" alt="image" src="https://user-images.githubusercontent.com/229881/172727309-98a0815d-c72e-4c50-b738-5c1b0daf4fdd.png">

Its just collapsed by default 
<img width="409" alt="image" src="https://user-images.githubusercontent.com/229881/172727379-21c68242-1c43-4738-8b03-541810d3bd81.png">


Makes sense! I think the comment needs to be updated because "not diffable" doesn't seem true.

Reply:
Updated the comment!